explorer/ExplorerControls.tsx

```diff
@@ -53,6 +53,56 @@ export class ExplorerControlBar extends React.Component<{
     }
 }
 
+interface ExplorerDropdownOption {
+    label: string
+    value: string
+}
+
+const ExplorerSingleValue = (props: SingleValueProps) => {
+    if (props.selectProps.isSearchable && props.selectProps.menuIsOpen)
+        return (
+            <components.SingleValue {...props}>
+                <span style={{ fontStyle: "italic", opacity: 0.75 }}>
+                    Type to search&hellip;
+                </span>
+            </components.SingleValue>
+        )
+    else return <components.SingleValue {...props} />
+}
```

Comment:
This is the new stuff. The rest is mostly factoring out the Select into its own component.